python/fedml/computing/scheduler/scheduler_entry/launch_manager.py

```diff
@@ -782,7 +774,8 @@ def __init__(self, job_yaml_file, should_use_default_workspace=False):
         self.base_dir = os.path.dirname(job_yaml_file)
         self.using_easy_mode = True
         self.executable_interpreter = "bash"
-        self.executable_file_folder = self.job_config_dict.get("workspace", None) \
+        self.executable_file_folder = os.path.normpath(
```

Comment:
If workspace is a absolute path, the self.executable_file_folder will be not correct.

Reply:
I think we have trade off to make:

1. With my modified implementation, workspace needs to always be relative path to `job.yaml`. The upside is, `fedml` commands can be ran from outside the directory and would still work. For eg, if `job.yaml` is in example folder (`example/job.yaml`), you need NOT require to run `fedml` from within example  directory and can run from anywhere as long as you give correct path to `job.yaml` i.e. `fedml launch /path/to/job.yaml`  would work.

2. With current implementation, workspace can be either relative or absolute path. But then, one has to always run `fedml` command from within the directory where `job.yaml` is present i.e. `fedml launch /path/to/job.yaml ` won't work. one would have to `cd` to `/path/to/job.yaml` and then execute `fedml launch job.yaml`

What do we think is more convenient to users?